scripts/globals/effects/yonin.lua

```diff
@@ -28,4 +35,10 @@ function onEffectLose(target,effect)
     target:delMod(dsp.mod.ACC,-effect:getPower())
     target:delMod(dsp.mod.NINJA_TOOL,effect:getPower())
     target:delMod(dsp.mod.ENMITY,effect:getSubPower())
+    if target:getMainJob() == dsp.job.NIN and target:getMainLvl() >= 75 then
```

Comment:
You don't need to check for main level here. getMerit does that for you.

Reply:
ah, I didn't go deep enough in the stack to see that. That's nice. sort of, now I gotta remove them all lol